mongo/src/main/java/com/redhat/lightblue/mongo/crud/MongoCRUDController.java

```diff
@@ -834,10 +834,12 @@ public EntityMetadata getEntityMetadata(String entityName) {
                     coll.save(doc);
                 } catch (Exception e) {
                     LOGGER.error(e.getMessage());
-                    LOGGER.error("Error saving doc:\n{}", doc);
+                    LOGGER.debug("Original doc:\n{}", original);
+                    LOGGER.debug("Error saving doc:\n{}", doc);
                 }
             }
         }
+        cursor.close();
```

Comment:
The cursor iteration block can use a try-catch-log-finally, so we can see errors if read fails.